Source/ocdm/open_cdm_ext.cpp

```diff
@@ -362,5 +364,30 @@ opencdm_construct_session(struct OpenCDMSystem* system,
     }
 
     TRACE_L1("Created a Session, result %p, %d", *session, result);
-    return (result);
+    return result;
+}
+
+OpenCDMError opencdm_system_ext_get_properties(struct PlayLevels* system, const char* propertiesJSONText) 
+{
+    ASSERT(system != nullptr);
+    OpenCDMError result(ERROR_INVALID_ACCESSOR);
+
+    if (system != nullptr) {
```

Comment:
You probably want to check propertiesJSONText for being a nullptr as well, constructing a std::string from a nullptr is undefined